xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererOpenGLES.cpp

```diff
@@ -246,8 +246,9 @@ void CRPRendererOpenGLES::DrawBlackBars()
 
 void CRPRendererOpenGLES::Render(uint8_t alpha)
 {
-  CRenderBufferOpenGLES* renderBuffer = static_cast<CRenderBufferOpenGLES*>(m_renderBuffer);
+  const ViewportCoordinates dest{m_rotatedDestCoords};
 
+  CRenderBufferOpenGLES* renderBuffer = static_cast<CRenderBufferOpenGLES*>(m_renderBuffer);
```

Comment:
Sonarqube will suggest here to use auto to avoid duplication of types

Reply:
I think the greatest value in sonarqube is not the code improvements, but your eye (and now mine) for such possible improvements before they're merged. Lets get this in with duplication, then bump to auto, then always do auto for duplicate types going forward.